code/AssetLib/Blender/BlenderScene.h

```diff
@@ -758,10 +756,7 @@ struct Scene : ElemBase {
 
     ListBase base;
 
-    Scene() :
-            ElemBase(), camera(), world(), basact(), master_collection() {
-        // empty
-    }
```

Comment:
base class default c’tors are always called; `std::shared_ptr` is always default-initialized